.github/workflows/integration-tests.yml

```diff
@@ -30,34 +30,8 @@ jobs:
           distribution: 'temurin'
           cache: 'gradle'
 
-      - name: Gradle build service
-        run: ./gradlew --build-cache :service:build -x test
-
-  jib:
```

Comment:
This change isn't directly related but I noticed that we're still running the `jib` build although the output of this operation isn't used.